doc/devguide/contributing.rst

```diff
@@ -35,7 +35,7 @@ please use our `Discussions page <https://github.com/arfc/saltproc/discussions>`
 Instructions for setting up a development environment
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The SaltProc is compatible with Python >=3.5. Anaconda is the recommended distribution to use to work on SAltProc; we will assume that if you want to use another distribution or your own set up, you can translate the instructions.
+The SaltProc is compatible with Python >=3.5. Anaconda is the recommended distribution to use to work on SaltProc; we will assume that if you want to use another distribution or your own set up, you can translate the instructions.
```

Comment:
Is this still true? Because Python 3.5 is no longer supported we may want to bump this.

Reply:
I agree. Let's bump it to 3.7?